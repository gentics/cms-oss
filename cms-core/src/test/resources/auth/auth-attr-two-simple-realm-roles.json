{
  "roles": {
    "realm": [
      "realm-role-1",
      "realm-role-2"
    ]
  }
}