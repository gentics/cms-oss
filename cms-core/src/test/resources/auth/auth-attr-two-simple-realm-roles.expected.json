{
  "Group1": null,
  "Group2": null,
  "4711": null
}